test/BabelRewirePluginTransformTest.js

```diff
@@ -84,6 +84,7 @@ describe('BabelRewirePluginTest', function() {
 		'noDefaultExport',
 		'primitiveExportWithNamedFunctionExport',
 		'wildcardImport',
+    'wildcardExport',
```

Comment:
Seems like this project uses tabs for whitespace. You might want to convert these indentations to use tabs.